packages/@office-iss/react-native-win32/src/Libraries/Components/Button.win32.js

```diff
@@ -328,6 +328,8 @@ class Button extends React.Component<ButtonProps> {
       disabled: ariaDisabled ?? accessibilityState?.disabled,
       expanded: ariaExpanded ?? accessibilityState?.expanded,
       selected: ariaSelected ?? accessibilityState?.selected,
+      multiselectable: accessibilityState?.multiselectable,
+      required: accessibilityState?.required,
```

Comment:
Are there no aria equivalents?

Reply:
aria-required and aria-multiselectable exist on the web but haven't been added to react-native (unlike the other aria- props here)

Reply:
Maybe not for this change, but given they're only implemented on windows/win32 as of now, should we just add them to RNW for now? Or add an issue to do so later?

Reply:
I'm not sure if these props were implemented for windows... I'm only certain about win32. But yeah, I can create an issue for it.

Reply:
https://github.com/microsoft/react-native-windows/issues/12047

Reply:
note: I took a look at it this morning and added the corresponding aria-props